horace_core/sqw/parse_pixel_indices.m

```diff
@@ -0,0 +1,76 @@
+function [irun, idet, ien] = parse_pixel_indices (win, varargin)
+% Return the run, detector and energy indices of pixels in one or more sqw objects
+%
+%   >> [irun, idet, ien] = parse_pixel_indices (win)        % all pixels
+%   >> [irun, idet, ien] = parse_pixel_indices (win, ipix)  % selected pixels
+%
+% Note: an efficient way of checking that ipix is consistent with all the sqw
+% objects is to call without output arguments i.e.
+%
+%   >> parse_pixel_indices (win, ipix)
+%
+% This will return silently without without spending time computing output
+% argument irun if the arguments are consistent, but will throw an error with an
+% appropriate message if e.g. ipix is out of range for
```

Comment:
"for" what?

Reply:
Yes... Chris spotted my hanging edit too. Comment changed.